resources/pipes-pipe.go

```diff
@@ -0,0 +1,76 @@
+package resources
+
+import (
+	"context"
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/pipes"
+
+	"github.com/ekristen/libnuke/pkg/registry"
+	"github.com/ekristen/libnuke/pkg/resource"
+	"github.com/ekristen/libnuke/pkg/types"
+
+	"github.com/ekristen/aws-nuke/v3/pkg/nuke"
+)
+
+const PipesPipeResource = "PipesPipes"
```

Comment:
We want the resource type to be singular. The plural on the service is fine, we usually try to make that match.

```suggestion
const PipesPipeResource = "PipesPipe"
```